### DISCUSSION TITLE
How can I obtain more data points for multiphase permeability curves

### DISCUSSION THREAD

**Discussion by ChenShiTong-CN** [PMEAL/OpenPNM#2593] [Q&A]
I used Stokes flow to simulate a two-phase fluid and obtained the relative permeability curve, but whether it was because of the large permeability of my model, the points of the phase permeability curve I obtained were concentrated at both ends, and the data points in the middle were very few, Is there anything I can do to improve the situation？

**Comment by jgostick** [comment]
Hi @ChenShiTong-CN , this is an easy one to answer...use InvasionPercolation.  It simulates the invasion of each pore individually, so you can get the fluid configutation for the entire range of saturations.  For a bit of discussion on this our paper on [imaged-based invasion percolation](https://doi.org/10.1016/j.cageo.2021.104978).

**Reply by ChenShiTong-CN** [reply]
I will try it right away. Thank you very much for your reply
